front/app/containers/Admin/settings/customize/index.tsx

```diff
@@ -729,7 +732,12 @@ class SettingsCustomizeTab extends PureComponent<
                 </ToggleLabel>
               </Setting>
             </SectionField>
-            <Outlet id="app.containers.Admin.settings.customize.headerSectionEnd" />
+            <Outlet
```

Comment:
Only changed this line. The rest is Eslint.

P.S. Don't we want to turn it off? :thinking: